README.md

```diff
@@ -6,6 +6,9 @@
 [![devDependency Status](https://david-dm.org/thelounge/lounge/dev-status.svg)](https://david-dm.org/thelounge/lounge?type=dev)
 
 # The Lounge
+<p align="center">
+	<img src="https://raw.githubusercontent.com/thelounge/thelounge.github.io/master/img/screenshots/example_css.png" width="550" />
```

Comment:
> Also, <nitpick>there is no trailing slash on img elements in HTML5 :D</nitpick>.

The spec doesn't say anything about the usage of self-closing tags for void elements though afaik. `<img />` and `<img>` are both valid and are equal.